python/pyrogue/utilities/prbs.py

```diff
@@ -35,6 +35,10 @@ def __init__(self, *, name, width=None, taps=None, **kwargs ):
         if taps is not None:
             self._prbs.setTaps(taps)
 
+        if stream is not None:
```

Comment:
Why?

Reply:
We can probably remove this whole feature. The thought was that a Prbs Device is sort of useless without a stream connected to it, so you might as well have a way to pass it one right when you make it. I don't actually use it anywhere in HPS or KPIX though.
I also made `PrbsPair` but then never ended up using it anywhere.